src/event_gate_lambda.py

```
@@ -80,17 +81,33 @@
+
+    if not raw_keys:
+        raise KeyError(f"No public keys found in {TOKEN_PUBLIC_KEYS_URL} endpoint response")
+
+    TOKEN_PUBLIC_KEYS: list[RSAPublicKey] = [
+        serialization.load_der_public_key(base64.b64decode(raw_key)) for raw_key in raw_keys
+    ]
+    logger.debug("Loaded %d TOKEN_PUBLIC_KEYS", len(TOKEN_PUBLIC_KEYS))
 except (requests.RequestException, ValueError, KeyError, UnsupportedAlgorithm) as exc:
-    logger.exception("Failed to fetch or deserialize token public key from %s", CONFIG.get("token_public_key_url"))
+    logger.exception("Failed to fetch or deserialize token public key from %s", TOKEN_PUBLIC_KEYS_URL)
     raise RuntimeError("Token public key initialization failed") from exc
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)